registry/storage/blobstore.go

```diff
@@ -85,6 +87,52 @@ func (bs *blobStore) Put(ctx context.Context, mediaType string, p []byte) (distr
 	}, bs.driver.PutContent(ctx, bp, p)
 }
 
+func (bs *blobStore) Create(ctx context.Context, options ...distribution.BlobCreateOption) (distribution.BlobWriter, error) {
```

Comment:
Wondering why `blobStore` needs to satisfy the whole `BlobService` interface. Why isn't `BlobEnumerator` enough?